### DISCUSSION TITLE
Changing the exposure time finder test model.

### DISCUSSION THREAD

**Discussion by MakManIfest** [sn4k3/UVtools#793] [Ideas]
This may already be possible and if it is please let me know.
But if it is not, I request the ability to change the model printed for the exposure time finder test.
I like to use J3D Techs Boxes of calibration model for dialing in the exposure time.

I was making custom RERF slices for my Anycubic machines but I have now upgraded to a Saturn 3 Ultra and need to find a new way to do the same thing.
I believe having the ability to choose the model you want to use for the exposure time test would be a useful feature to many people.

**Comment by sn4k3** [comment]
On the dialog, scroll down where it says "Pattern loaded model" that allow you to use the current loaded model as test instead of the blocks

**Reply by MakManIfest** [reply]
Aww sick! Thank you very much. Didn't see that. I may have an older version of UV tools.

Edit: more likely I didn't understand what it meant lol. I will have a tinker when I finish work.
